site/source/docs/getting_started/downloads.rst

```diff
@@ -201,3 +189,38 @@ The entire Emscripten SDK is also available in the form of a `docker image
     emscripten/emsdk emcc helloworld.cpp -o helloworld.js
 
 See the Docker Hub page for more details and examples.
+
+Installation using packages
+=========================
+
+.. note:: the `emsdk` is the only officially supported way to use
+    Emscripten that is supported by the Emscripten project, and the only one
+    that we constantly test
+    (`emsdk CI <https://github.com/emscripten-core/emsdk/blob/master/.circleci/config.yml>`_,
+    `Emscripten GitHub CI <https://github.com/emscripten-core/emscripten/blob/master/.circleci/config.yml>`_,
+    `Chromium CI <https://ci.chromium.org/p/emscripten-releases>`_). (While we
+    don't officially support other ways of getting Emscripten, we definitely
+    appreciate the efforts by third parties to
+    `package Emscripten <https://github.com/emscripten-core/emscripten/blob/master/docs/packaging.md>`_
+    for users' convenience, and we'd like to help out, please get in touch if
+    you are such a packager!)
+
+The following is a partial list of downstream emscripten packages. While these
+packages are not officially supported by the emscripten developers we try to
+work with downstream packagers to ensure everything works as expected.
+
+**Windows**
+- package info: `emscripten` in [chocolatey](https://chocolatey.org/packages/emscripten)
+- maintainer: @aminya
+
+**Homebrew**
+ - package info: https://formulae.brew.sh/formula/emscripten
+ - maintainer: @chenrui333
+
+**Arch Linux**
+ - package info: https://github.com/archlinux/svntogit-community/tree/packages/emscripten/trunk
+ - maintainer: Sven-Hendrik Haase <svenstaro@gmail.com>
+
+**Gentoo Linux** (custom overlay)
+ - package info: `dev-util/emscripten` in [darthgandalf-overlay](https://github.com/DarthGandalf/gentoo-overlay)
+ - maintainer: @DarthGandalf
```

Comment:
I wonder if it makes sense to duplicate this  information here.   I originally put this info in packaging.md so we have somewhere to internally coordinate with packagers .. but I can see that this is could be useful to end users too, although perhaps not all of it.

If we do move it here we should probably remove it from the other location and replace that with a link.

Reply:
I can remove it from the packaging.md

Reply:
Fixed in https://github.com/emscripten-core/emscripten/pull/13149/commits/6fba55b61bf4c1f7930ca320a2a5c88b6377edd3